tests/framework/results/helpers.py

```diff
@@ -74,7 +74,6 @@ def columns_created(self) -> list[str]:
     def setup(self, builder: Builder) -> None:
         builder.value.register_attribute_producer(
             "grade",
-            # FIXME [MIC-6631]: Allow source to be AttributePipelines directly
```

Comment:
These are just kinda weird b/c it's actually mutating the pipeline return so I just kept it in a lambda